src/openscm_runner/adapters/fair_adapter/fair_adapter.py

```diff
@@ -48,17 +48,21 @@ def _make_full_cfgs(self, scenarios, cfgs):  # pylint: disable=R0201
             emissions_pi[8] = 3.8773253867471933
             emissions_pi[9] = 2.097770755
             emissions_pi[10] = 15.44766815
+            emissions_pi[11] = 6.92769009144426
+
+            nt = emissions.shape[0]
+            # TODO: raise error if nt > 736
```

Comment:
Do you want to fix this before merging?

Reply:
Actually let's just delete as an error will be thrown further down the code and I don't want to complicate the possibility of running sub-yearly.